modules/administration/pages/tshoot-osadjabberd.adoc

```diff
@@ -23,35 +23,52 @@ If more detailed instructions are required, put them in a "Resolving" procedure:
 
 == Open File Count Exceeded
 
-The number of maximum files that a jabber user can open is lower than the number of connected clients.
-OSAD clients cannot contact the SUSE Manager Server, and jabberd requires long periods of time to respond on port 5222.
-Each client requires one permanently open TCP connection and each connection requires one file handler.
+In some cases, the maximum number of files that jabber can open is lower than the number of connected OSAD clients.
 
-To resolve OSAD connection problems, edit such lines in [path]``/etc/security/limits.conf``:
+If this occurs, OSAD clients cannot contact the SUSE Manager Server, and jabberd will take an excessive amount of time to respond on port 5222.
 
-----
-jabber   soft   nofile   <#clients + 100>
-jabber   hard   nofile   <#clients + 1000>
-----
+[[NOTE]]
+====
+This fix is only required if you have more than 8192 clients connected using OSAD.
+In this case, we recommend you consider using Salt clients instead.
+For more information about tuning large scale installations, see xref:salt:large-scale.adoc[].
+====
+
+You can increase the number of files available to jabber by editing the [systemitem]``LimitNOFILE`` service file in [command]``systemd``.
 
-Itwill vary according to your setup.
-For example, in the case of 5000 clients:
+.Procedure: Adjusting the Maximum File Count
 
+. As root, edit the [path]``/etc/systemd/system/httpd.service.d/local.conf`` file.
```

Comment:
I think the path is wrong. As we talk about the jabberd.service we should modify that and not a httpd service.
Or I am totally wrong?

Reply:
I don't know. I'm not sure where this originally came from. It would make sense to me that it's a jabberd service, though. Who can confirm?

Reply:
The service runs at `/usr/lib/systemd/system/jabberd.service`

Cannot for the life of me work out the systemd config file location for it, though. My best guess is `/usr/lib/systemd/system/jabberd.service/local.conf` but it's just a guess ...

Reply:
I am increasingly thinking that delinking this whole section is (after all) the smartest thing to do, you know ...